aikau/src/main/resources/alfresco/forms/controls/MultiSelect.js

```diff
@@ -0,0 +1,1230 @@
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// TODO: Add ARIA
+
+/**
+ * @module alfresco/forms/controls/MultiSelect
+ * @extends external:dijit/_TemplatedMixin
+ * @extends external:dijit/_WidgetBase
+ * @author Martin Doyle
+ */
+define([
+      "alfresco/core/Core",
+      "alfresco/core/ObjectTypeUtils",
+      "dijit/_FocusMixin",
+      "dijit/_TemplatedMixin",
+      "dijit/_WidgetBase",
+      "dojo/_base/array",
+      "dojo/_base/declare",
+      "dojo/_base/lang",
+      "dojo/Deferred",
+      "dojo/dom-construct",
+      "dojo/dom-style",
+      "dojo/dom-class",
+      "dojo/keys",
+      "dojo/on",
+      "dojo/when",
+      "dojo/text!./templates/MultiSelect.html"
+   ],
+   function(Core, ObjectTypeUtils, _FocusMixin, _TemplatedMixin, _WidgetBase, array, declare, lang, Deferred, domConstruct, domStyle, domClass, keys, on, when, template) {
+
+      return declare([_WidgetBase, _TemplatedMixin, _FocusMixin, Core], {
+
+         /**
+          * The Choice object (referenced in other JSDoc comments)
+          *
+          * @instance
+          * @typedef {object} Choice
+          * @property {object} domNode The main domNode for the choice
+          * @property {object} contentNode The content domNode inside the choice
+          * @property {object} closeButton The domNode of the close button
+          * @property {object} selectListener A remove handle for the choice selection listener
+          * @property {object} closeListener A remove handle for the close-button listener
+          * @property {object} item The store item
+          * @property {string} label The label for this choice
+          * @property {string} value The value of this choice
+          */
+
+         /**
+          * The Result object (referenced in other JSDoc comments)
+          *
+          * @instance
+          * @typedef {object} Result
+          * @property {object} domNode The main domNode for the result
+          * @property {object} item The store item
+          * @property {string} label The label for this choice
+          * @property {string} value The value of this choice
+          */
+
+         /**
+          * An array of the CSS files to use with this widget.
+          *
+          * @instance
+          * @type {object[]}
+          * @default [{cssFile:"./css/MultiSelect.css"}]
+          */
+         cssRequirements: [{
+            cssFile: "./css/MultiSelect.css"
+         }],
+
+         /**
+          * An array of the i18n files to use with this widget.
+          *
+          * @instance
+          * @type {object[]}
+          * @default [{i18nFile: "./i18n/MultiSelect.properties"}]
+          */
+         i18nRequirements: [{
+            i18nFile: "./i18n/MultiSelect.properties"
+         }],
+
+         /**
+          * The localised loading message used by the template
+          *
```

Comment:
Need @instance here (and on all the other attributes too, otherwise it gets displayed as a global attribute in the final JSDoc